www/src/pages/en/t3-collection.md

```diff
@@ -8,13 +8,14 @@ Made a project using the T3 stack and want to share it? Add it to the list!
 
 ## Open Source apps made using the T3 Stack
 
-| Description                                                      | Repo                                                                  | Link                               |
-| ---------------------------------------------------------------- | --------------------------------------------------------------------- | ---------------------------------- |
-| Zapdos - a QnA app for streamers                                 | [t3-oss/zapdos](https://github.com/t3-oss/zapdos)                     | [z.t3.gg](https://z.t3.gg)         |
-| nexxel's personal site                                           | [nexxeln/nexxel.dev](https://github.com/nexxeln/nexxel.dev)           | [nexxel.dev](https://nexxel.dev)   |
-| Julius's personal site                                           | [juliusmarminge/jumr.dev](https://github.com/juliusmarminge/jumr.dev) | [jumr.dev](https://jumr.dev)       |
-| Me3 - Describe yourself in 3 things and share with your friends. | [hnqg/me3](https://github.com/hnqg/me3)                               | [me.hnqg.io](https://me.hnqg.io)   |
-| Josh's personal site                                             | [joshhyde9/portfolio](https://github.com/JoshHyde9/portfolio)         | [joshhyde.me](https://joshhyde.me) |
+| Description                                                      | Repo                                                                  | Link                                 |
+| ---------------------------------------------------------------- | --------------------------------------------------------------------- | ------------------------------------ |
+| Zapdos - a QnA app for streamers                                 | [t3-oss/zapdos](https://github.com/t3-oss/zapdos)                     | [z.t3.gg](https://z.t3.gg)           |
+| Shoutify - Free, open-source, sel-hosted social media management | [techsquidtv/shoutify](https://github.com/TechSquidTV/Shoutify)       | [shoutify.app](https://shoutify.app) |
```

Comment:
I know you said on the other PR that the site is coming soon, but I'm not sure we should add a link to the docs that is currently dead - happy to add it once the site is up.